src/DotNetCore.CAP/Internal/ICapPublisher.Default.cs

```diff
@@ -38,6 +38,11 @@ public CapPublisher(IServiceProvider service)
 
         public AsyncLocal<ICapTransaction> Transaction { get; }
 
+        public void Start(CancellationToken stoppingToken)
```

Comment:
Should it be removed?  I think the prevent Start method in CapPublisher looks a bit strange?  and the method is not called

Reply:
It does look strange on the publisher side agreed. I need to come back to this and reassess. It didn't quite fit the current structure.